### ISSUE TITLE
`yarn bin` doesn't do the same thing w/ yarn berry

### ISSUE THREAD

**Issue by jsoref** [square/find-yarn-workspace-root#50]
https://github.com/square/find-yarn-workspace-root/blob/0e99142b19bbbb0b9e72b7ff72e125b61d7e89d6/README.md?plain=1#L46-L50

https://yarnpkg.com/cli/bin/#examples

> List all the available binaries :
>
> ```
> yarn bin
> ```

I think the yarn 3 equivalent would be:

```bash
$(yarn bin standard-version)
```

**Comment by bmish** [comment]
Yeah those release commands may be a bit outdated. But note that this is only for the internal commands used by maintainers.

**Comment by jsoref** [comment]
I'd encourage you to move those items out of the main `README.md` as its contents are displayed in the npmjs view...
